evm/hyperliquid.js

```diff
@@ -0,0 +1,206 @@
+    const [tokenId, deployer] = args;
+    validateParameters({
+        isNonEmptyString: { tokenId },
+        isValidAddress: { deployer },
+    });
+
+    const contracts = chain.contracts;
+    const interchainTokenServiceAddress = contracts.InterchainTokenService?.address;
+
+    validateParameters({
+        isValidAddress: { interchainTokenServiceAddress },
+    });
+
+    const IInterchainTokenService = getContractJSON('IInterchainTokenService');
+    const interchainTokenService = new Contract(interchainTokenServiceAddress, IInterchainTokenService.abi, wallet);
+
+    const tokenAddress = await interchainTokenService.registeredTokenAddress(tokenId);
+    printInfo('Token address', tokenAddress);
+
+    const InterchainTokenService = getContractJSON('HyperliquidInterchainTokenService');
+    const service = new Contract(interchainTokenServiceAddress, InterchainTokenService.abi, wallet);
+
+    const HyperliquidInterchainToken = getContractJSON('HyperliquidInterchainToken');
+    const token = new Contract(tokenAddress, HyperliquidInterchainToken.abi, wallet);
+
+    const currentDeployer = await token.deployer();
+    printInfo('Current deployer', currentDeployer);
+    printInfo('New deployer', deployer);
+
+    const serviceOwner = await service.owner();
+    const isOperator = await service.isOperator(wallet.address);
+
+    if (wallet.address.toLowerCase() !== serviceOwner.toLowerCase() && !isOperator) {
+        throw new Error('Wallet does not have permission to update deployers. Must be service owner or operator.');
+    }
+
+    const gasOptions = await getGasOptions(chain, options, 'HyperliquidInterchainTokenService');
+
+    const tx = await service.updateTokenDeployer(tokenId, deployer, gasOptions);
+    await handleTx(tx, chain, service, 'updateTokenDeployer');
+
+    const updatedDeployer = await token.deployer();
+    printInfo('Updated deployer', updatedDeployer);
+}
+
+async function main(processor, args, options) {
+    return mainProcessor(options, (config, chain, options) => {
+        if (!isHyperliquidChain(chain)) {
+            throw new Error(`Chain "${chain.name}" is not supported. This script only works on Hyperliquid chains.`);
+        }
+
+        const rpc = chain.rpc;
+        const provider = getDefaultProvider(rpc);
+        const wallet = new Wallet(options.privateKey, provider);
+
+        return processor(wallet, chain, args, options);
+    });
+}
+
+async function switchHyperliquidBlockSize(options, useBigBlocks, chain) {
+    const blockType = useBigBlocks ? 'big' : 'small';
+    const rpc = chain.rpc;
+    const provider = getDefaultProvider(rpc);
+    const wallet = new Wallet(options.privateKey, provider);
+    await updateBlockSize(wallet, chain, [blockType]);
+}
+
+if (require.main === module) {
+    const program = new Command();
+
+    program.name('hyperliquid').description('Hyperliquid chain management commands');
+
+    program
+        .command('update-block-size <blockSize>')
```

Comment:
```suggestion
        .command('update-block-size <block-size>')
```
we use kebab case for the new cli syntax